etcd/fs/fs.go

```diff
@@ -341,6 +341,18 @@ func (obj *Fs) Create(name string) (afero.File, error) {
 	return fileCreate(obj, name)
 }
 
+// Chown is the equivalent of os.Chown. It returns ErrNotImplemented.
+func (obj *Fs) Chown(name string, uid, gid int) error {
+	// FIXME Implement Chown
```

Comment:
This is all fantastic, one small nit: we have a test that checks for these in the form of `FIXME: The message` (so please add the colon so it doesn't fail here and below)

Reply:
Ack, done.